content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: कुबेरनेट्स प्रलेखन  
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "होम"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन"
+    weight: 20
+    post: >
+      <p>वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन  के साथ कुबेरनेट्स का उपयोग करना सीखें| आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान करने में</a>! भी मदद कर सकते हैं</p>
+description: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है| यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है।
+overview: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
+cards:
+- name: concepts
+  title: "मूल बातें समझें"
+  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें|"
+  button: "अवधारणाओं को जानें"
+  button_path: "/hi/docs/concepts"
+- name: tutorials
+  title: "कुबेरनेट्स को उपयोग कर ने का प्रयास करें"
+  description: "कुबेरनेट्स में एप्लिकेशन को परिनियोजित करने का तरीका जानने के लिए ट्यूटोरियल का अनुसरण करें|"
+  button: "ट्यूटोरियल देखें"
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेट करें"
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+  button: "कुबेरनेट्स को सेट करें "
+  button_path: "/hi/docs/setup"
+- name: tasks
+  title: "कुबेरनेट्स का उपयोग करना सीखें"
+  description: "सामान्य कार्यों को देखें और चरणों के एक छोटे क्रम का उपयोग करके उन्हें करें|"
+  button: "कार्य देखें"
+  button_path: "/hi/docs/tasks"
+- name: training
+  title: "प्रशिक्षण"
+  description: "कुबेरनेट्स में प्रमाणित हो जाएं और अपने क्लाउड नेटिव प्रोजेक्ट को सफल बनाएं!"
+  button: "प्रशिक्षण देखें"
+  button_path: "/training"
+- name: reference
+  title: संदर्भ जानकारी देखें 
+  description: शब्दावली, कमांड लाइन सिंटैक्स, एपीआई संसाधन प्रकार और सेटअप टूल दस्तावेज़ खोजें|
+  button: संदर्भ देखें 
+  button_path: /hi/docs/reference
+- name: contribute
+  title: प्रलेखन में योगदान करें 
+  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हों या आप लंबे समय से हैं|
+  button: प्रलेखन में योगदान करें 
+  button_path: /hi/docs/contribute
```

Comment:
```suggestion
  button_path: /docs/contribute
```
unless `/hi/docs/contribute` is already there to link to.

Reply:
Oh, actually i was following the `Korean` localization, so according to that i edited the link!
Sure, we can remove that for now 🙂